main.js

```diff
@@ -3,14 +3,282 @@ const close = document.getElementById('close');
+    language1: 'html',
+    language2: 'css',
+    language3: 'javaScript',
+    language4: 'github',
+    language5: 'ruby',
+    language6: 'bootstrap',
+    liveBtn: 'https://normainobary2021.github.io/',
+    sourceBtn: 'https://github.com/normainobary2021/my-portfolio',
+  },
+];
+
+[menuIcon, close, item].forEach((e) => {
+  e.addEventListener('click', () => {
+    menu.classList.toggle('hidden');
+    menu.classList.toggle('hidden');
+    menu.classList.toggle('hidden');
+  });
+});
+
+let j = 0;
+
+function popup() {
+  const popupScreen = document.querySelector('body');
+  const popupDiv = document.createElement('div');
+  const wrapperDiv = document.createElement('div');
+  const wrapperHeadDiv = document.createElement('div');
+  const divTitle = document.createElement('h2');
+  const divSpan1 = document.createElement('span');
+  const wrapperSubHeadDiv = document.createElement('div');
+  const workSubText = document.createElement('p');
+  const dotSpan1 = document.createElement('span');
+  const descrSpan1 = document.createElement('span');
+  const dotSpan2 = document.createElement('span');
+  const descrSpan2 = document.createElement('span');
+  const wrapperImageDiv = document.createElement('div');
+  const workImageDiv = document.createElement('div');
+  const imageItem1 = document.createElement('img');
+  const wrapperTextDiv = document.createElement('div');
+  const wrapperTextP1 = document.createElement('p');
+  const wrapperTextP2 = document.createElement('p');
+  const p2Span1 = document.createElement('span');
+  const p2Span2 = document.createElement('span');
+  const p2Span3 = document.createElement('span');
+  const hr = document.createElement('hr');
+  const wrapperBtnDiv = document.createElement('div');
+  const wrapperBtn1 = document.createElement('a');
+  const wrapperIBtn1 = document.createElement('i');
+  const wrapperIBtn2 = document.createElement('i');
+  const wrapperBtn2 = document.createElement('a');
+  const fullModalDiv = document.createElement('div');
+  const fullImgDiv = document.createElement('div');
+  const fullDivImg = document.createElement('img');
+  const fullMdlTxtDiv = document.createElement('div');
+  const fullMdlP1 = document.createElement('p');
+  const fullMdlP2 = document.createElement('p');
+  const fullSpan1 = document.createElement('span');
+  const fullSpan2 = document.createElement('span');
+  const fullSpan3 = document.createElement('span');
+  const fullMdlP3 = document.createElement('p');
+  const fullSpan4 = document.createElement('span');
+  const fullSpan5 = document.createElement('span');
+  const fullSpan6 = document.createElement('span');
+  const seeLiveBtn = document.createElement('button');
+  const seeLiveI = document.createElement('i');
+  const seeSourceBtn = document.createElement('button');
+  const seeSourceI = document.createElement('i');
+
+  popupDiv.className = 'overlay';
```

Comment:
- [ ]  Your button should have the size similar to the one from the Figma design
![image](https://user-images.githubusercontent.com/43877689/180429709-60364401-707e-4853-b740-bb5c694a5459.png)
![image](https://user-images.githubusercontent.com/43877689/180429987-48f80dc7-2deb-4be1-a1ad-28d43b081bc5.png)